Gopkg.lock

```diff
@@ -24,6 +24,22 @@
   revision = "3e01752db0189b9157070a0e1668a620f9a85da2"
   version = "v1.0.6"
 
+[[projects]]
```

Comment:
Ouch, so we only vendor in to get the CI to pass, and we vendor into the top level dir?
Any chance we can vendor this under `kata-webhook/vendor` maybe?

Reply:
Agreed - for an example of that "sub-vendoring", see: https://github.com/kata-containers/tests/tree/master/cmd/log-parser

Reply:
Done and also switched the docker build to use the vendored files. No sense wasting time in the build process when we have already have vendored dependencies available.